src/main/java/org/apache/sysml/api/python/SystemML.py

```diff
@@ -25,10 +25,327 @@
 import os
 from pyspark.sql import DataFrame, SQLContext
 from pyspark.rdd import RDD
+from contextlib import contextmanager
+import numpy as np
+import pandas as pd
 
+def simpleAddPyDMLStmt(fnName, args=None, obj=None):
+    m = Matrix(0)
+    addPyDMLStmt(fnName, lhs=m, args=args, obj=obj)
+    return m
 
-class MLContext(object):
+def addPyDMLStmt(fnName, lhs=None, args=None, obj=None):
+    pydmlStmt = [getTabs()]
+    if lhs is not None:
+        pydmlStmt = pydmlStmt + [str(lhs.getID()), ' = ']
+    if obj is None:
+        pydmlStmt = pydmlStmt + [ fnName, '(']
+    else:
+        pydmlStmt = pydmlStmt + [  obj, '.', fnName, '(']
+    if args is not None:
+        pydmlStmt = pydmlStmt + args
+    MLContext.globalMLContext.pydml.append(''.join(pydmlStmt + [')\n']))
+    
+# Constructor         
+def full(init, rows=None, cols=None):
+    if rows is None or cols is None:
+        raise Exception('rows and cols are mandatory arguments')
```

Comment:
Better yet, we should just copy the syntax of [`numpy.full`](http://docs.scipy.org/doc/numpy/reference/generated/numpy.full.html).